README.md

```diff
@@ -1,11 +1,28 @@
 # Driver Drowsiness Detection System
 
 ## Description
+<<<<<<< HEAD
+This is a drowsiness detection system.In the program, we check if the eyes are open or closed for a certain period of time. If closed, the program will alert the user playng an alarm sound.
+
+##Applications
+This can be used to determine if a driver is drunk
+
+##Requirements
+nose
+tensorflow
```

Comment:
We are not using tensorflow for this project.
We used pytorch

Reply:
Ok I will rectify it